app/components/_vars.scss

```diff
@@ -1,7 +1,34 @@
+// colors
+$conversationDarkGray: #222;
+$appDefaultBlue: #007be3;
+$appFusionYellow: #ffc533;
+$appFusionTeal: #2ac3be;
+$appFusionRed: #f2726f;
+$appFusionPurple: #5d62b5;
+
+// Layout
+.vertical-center {
+    margin: 0;
+    position: relative;
+    top: 50%;
+    -ms-transform: translateY(-50%);
+    transform: translateY(-50%);
+}
```

Comment:
do we use vertical-center anywhere? If we do, it should be using flex to center things vertically instead of top: 50%.

Reply:
turns out I don't use this style. removed